src/include/disks.h

```diff
@@ -179,6 +179,7 @@ struct disk {
   fatfs_t *fatfs;		/* for FAT file system emulation */
   int mfs_idx;
   int part_image;               /* partition image */
+  int genmbr;
```

Comment:
Do you really need both, as they
seem to be the same thing?
Also genmbr sounds like a bad name.
I realize you meant "generate mbr",
but that can also be "generic mbr",
which is actually not there.

Reply:
> Do you really need both, as they
seem to be the same thing?

Probably not, I'll look into it.

> I realize you meant "generate mbr",
but that can also be "generic mbr",
which is actually not there.

Mmm, I didn't think of that. Let's see if I can delete it, otherwise I'll rename.